src/renderer.ts

```diff
@@ -156,6 +158,632 @@ export class DefaultCellRenderer implements ICellRendererFactory {
 /**
  * a renderer rendering a bar for numerical columns
  */
+
+class HeatmapCellRenderer extends DefaultCellRenderer {
+
+  createSVG(col: Column, context: IDOMRenderContext): ISVGCellRenderer {
+    function cell_dim(total, cells) {
+      return (total / cells);
+    }
+
+    const total_width = col.getWidth();
+    const bins = (<any>col.desc).datalength;
+    const defaultdomain = [0, 100];
+    var domain = (<any> col.desc).domain;
+    domain = (domain === undefined || null) ? defaultdomain : domain;
+    const min = domain[0], max = domain[1];
+    var colorrange = (<any> col.desc).colorrange;
+    const defaultcolor = ['blue', 'red'];
```

Comment:
avoid using default values in the renderer, a better place is the model, since e.g., the colorrange is model dependent